tests/foreman/sys/test_fam.py

```diff
@@ -0,0 +1,42 @@
+"""Test class foreman ansible modules
+
+:Requirement: Other
+
+:CaseAutomation: Automated
+
+:CaseLevel: System
+
+:TestType: Functional
+
+:CaseImportance: Medium
+
+:Upstream: No
+"""
+from robottelo import ssh
+from robottelo.constants import FOREMAN_ANSIBLE_MODULES
+from robottelo.decorators import destructive
+from robottelo.decorators import run_in_one_thread
+
+
+@destructive
+@run_in_one_thread
+def test_positive_ansible_modules():
```

Comment:
```suggestion
def test_positive_ansible_modules_installation():
```
?

At the end of the day, we don't check ansible modules themselves, we check if they can be installed and that there is something working after installation.